ElectronClient/app/gui/MainScreen.jsx

```diff
@@ -105,13 +112,13 @@ class MainScreenComponent extends React.Component {
 
 		if (command.name === 'newNote') {
 			if (!this.props.folders.length) {
-				bridge().showErrorMessageBox(_('Please create a notebook first.'));
+				bridge().showErrorMessageBox(_('Please create a folder first.'));
 			} else {
 				await createNewNote(null, false);
 			}
 		} else if (command.name === 'newTodo') {
 			if (!this.props.folders.length) {
-				bridge().showErrorMessageBox(_('Please create a notebook first'));
+				bridge().showErrorMessageBox(_('Please create a folder first'));
```

Comment:
Sorry just to be clear: internally, in code, we use the term "folder" everywhere. In strings visible by the user however we use "Notebook", so please revert these two changes.